salsah/src/public/less/search.less

```diff
@@ -93,6 +93,9 @@ table.searchres td {
   	-khtml-box-shadow: 5px 5px 5px #222;
 	box-shadow: 5px 5px 5px #222;
 }
+.searchinfowin table {
+    color: white;
+}
 /*////////////////////////////////////////////////////////////////////////////*/
```

Comment:
fixes bug #156 , the original fix was done on the CSS file and has been overwritten when it was regenerated from the LESS source file.